### PR TITLE
fix: Treat "no data" value as a decimal number TDE-1077

### PR DESCRIPTION
**Pull request by l0b0**
#### Modification

It's represented as a decimal number string in the `gdalinfo -json`
output, so we should treat it as such in the code. This should detect
any values not numerically equal to -9999.

#### Checklist

- [x] Tests updated
- [ ] Docs updated (N/A)
- [x] Issue linked in Title

### CODE REVIEW

**Review by paulfouquet — APPROVED** [review]

File: scripts/files/file_tiff.py [hunk 1]
```diff
@@ -1,11 +1,14 @@
 import json
+from decimal import Decimal
 from enum import Enum
 from typing import Any, Dict, List, Optional
 from urllib.parse import unquote
 
 from scripts.gdal.gdal_helper import GDALExecutionException, gdal_info, run_gdal
 from scripts.gdal.gdalinfo import GdalInfo
 
+DEFAULT_NO_DATA_VALUE = Decimal(-9999)
```
Comment (paulfouquet) [inline scripts/files/file_tiff.py:10]: Not part of this PR but if you want you could add a comment about why we are using `-9999`

Comment (l0b0) [reply scripts/files/file_tiff.py:10]: Can do! Do you have a reference? I think it was mentioned recently, but I can't find it in Slack.

Comment (l0b0) [reply scripts/files/file_tiff.py:10]: #901 is in draft, because it depends on this PR.

**Review by blacha — APPROVED** [review]
